bazel/cython_library.bzl

```diff
@@ -47,7 +50,7 @@ def pyx_library(name, deps = [], py_deps = [], srcs = [], **kwargs):
             # Optionally use PYTHON_BIN_PATH on Linux platforms so that python 3
             # works. Windows has issues with cython_binary so skip PYTHON_BIN_PATH.
             cmd =
-                "PYTHONHASHSEED=0 $(location @cython//:cython_binary) --cplus $(SRCS) --output-file $(OUTS)",
+                "PYTHONHASHSEED=0 $(location @cython//:cython_binary) {} --cplus $(SRCS) --output-file $(OUTS)".format(include_flags),
```

Comment:
Is this feature addition essential to this PR? It doesn't seem to be used directly?

Reply:
Good catch. This was used in an earlier version of the new `grpc_tools` build. Removed.